cmd/snap/cmd_run.go

```diff
@@ -950,6 +974,8 @@ func (x *cmdRun) runSnapConfine(info *snap.Info, securityTag, snapApp, hook stri
 		return x.runCmdUnderGdb(cmd, env)
 	} else if x.useStrace() {
 		return x.runCmdUnderStrace(cmd, env)
+	} else if x.Ltrace != "no-ltrace" && x.ParserRan == 1 { // FIXME: why ParserRan == 1 ?
```

Comment:
We have ParserRan defined because without it, all commands would have been run under strace, or in this case ltrace if didn't include it. See 7e661add47dfd2edbd08a4164fd371792448bc87 for more details.